src/main/java/com/mael/ttt/Board.java

```diff
@@ -77,4 +77,12 @@ public int getIndexFromCoords(int row, int col) {
     public boolean isCellBusy(int index) {
         return !(board[getRowFromIndex(index)][getColFromIndex(index)] == EMPTY);
     }
+
+    public List<String> getValidIndexes() {
```

Comment:
The reason why I was not checking with the empty cells is because at the beginning I thought it would be a good idea to have more precise feedback. I removed that now.

Regarding the mismatch, the class returns an integer, yes, but it receives a string. It's the human input after all. It's explained [here](http://octopusonalightbeam.tk/journal/2016/01/21/day071.html).